### PR TITLE
Add Issue Form for Meta/Repository

### PR DESCRIPTION
**Pull request by errorline99**
tell me if there is something wrong i hope i did it right (this is my first open source contribution )

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: meta/repository Report
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
+labels: ["updates"]
+body:
+  - type: markdown
+    attributes:
+      value: '## Thanks for taking the time to contribute to the repository.'
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:8]: I'd like for there to be a checkbox here which asks the user to confirm that they  have searched for similar issues.
You can ofc, have this in addition to what you wrote already if you want that :) Always nice to thank contributors!
```suggestion
        value: '## Before you continue, please search our open/closed issues to see if a similar issue has been addressed.'

  - type: checkboxes
    attributes:
      label: I have searched through the issues and didn't find my problem.
      options:
        - label: Confirm
          required: true
```

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: meta/repository Report
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:1]: Can we add this emoji here?
```suggestion
name: 📇 Meta/Repository Feature
```

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:3]: This is the title of the issue the user is making.

I like to put in a suggestion here which goes something like this:
```suggestion
title: "📇 <feature name>"
```

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+labels: ["updates"]
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:4]: We don't have an update label, but we do have a label for 📇 Meta/Repository
```suggestion
labels: ["📇 Meta/Repository"]
```

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    id: additions
+    attributes:
+      label: changes
+      description: is there something unclear with the files ?.
+
+  - type: textarea
+    id: extrainformation
+    attributes:
+      label: Additional information
+      description: is there anything else that you like to comment a little on
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:27]: Can we change it to this?
```suggestion
      description: is there anything else that you'd like to add?
```

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:28]: We can remove the extra line breaks here :)

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    id: description
+    attributes:
+      label: additions
+      description: what type of additions do you think will make the repository more easily assessible.
+
+  - type: textarea
+    id: additions
+    attributes:
+      label: changes
+      description: is there something unclear with the files ?.
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:21]: Not sure what exactly you mean about this. Could you explain?

File: .github/ISSUE_TEMPLATE/confiq.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+name: meta/repository Report
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
+labels: ["updates"]
+body:
+  - type: markdown
+    attributes:
+      value: '## Thanks for taking the time to contribute to the repository.'
+
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/confiq.yml:9]:
```suggestion
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Hi this is Great!
Welcome to the open source community, and thank you very much for contributing!

This is mostly looking good (especially for your first contribution)!
I've suggested a few changes, and also some other comments:

1. Could you change the filename to something more appropriate? Maybe `meta_repository.yml`
2. Add `fix #42` to your commit so the issue is automatically closed when this is merged. You can use `git commit --amend` and then change the message to include `fix #42`

File: .github/ISSUE_TEMPLATE/confiq.yml (hunk 7 above)
Comment (errorline99) [reply .github/ISSUE_TEMPLATE/confiq.yml]: incase there is something wrong or confusing in the readme docs

**Comment by errorline99** [comment]
I think i changed everything you asked but still not sure about the git commit and how to do it

File: .github/ISSUE_TEMPLATE/confiq.yml (hunk 7 above)
Comment (Snailedlt) [reply .github/ISSUE_TEMPLATE/confiq.yml]: Okey. Could we perhaps just write something like
`Is there anything else we should know about?`

File: .github/ISSUE_TEMPLATE/meta_repository.yml [hunk 9]
```diff
@@ -0,0 +1,37 @@
+name: "📇 <feature name>"
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
+labels: ["📇 Meta/Repository"]
+body:
+  - type: markdown
+    attributes:
+      value: '## Thanks for taking the time to contribute to the repository.'
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/meta_repository.yml:8]: I don't think it works with multiple value fields here. Try this instead
```suggestion
      value: '## Thanks for taking the time to contribute to the repository.'
  
  - type: markdown
    attributes:   
```

So the whole code snippet should look like this:
```yml
  - type: markdown
    attributes:
      value: '## Thanks for taking the time to contribute to the repository.'
  
  - type: markdown
    attributes:
      value: '## Before you continue, please search our open/closed issues to see if a similar issue has been addressed.'
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
There's still a few things missing.
Go throuigh the suggestions I made earlier one by one, and click on `resolve` once you think you've resolved them

File: .github/ISSUE_TEMPLATE/meta_repository.yml (hunk 9 above)
Comment (errorline99) [reply .github/ISSUE_TEMPLATE/meta_repository.yml:8]: ohhh that makes a lot of sense thank for the tip

**Comment by errorline99** [comment]
made the change. really hope it looks good now

File: .github/ISSUE_TEMPLATE/confiq.yml (hunk 2 above)
Comment (Snailedlt) [reply .github/ISSUE_TEMPLATE/confiq.yml]: @errorline99
This one isn't resolved yet

File: .github/ISSUE_TEMPLATE/meta_repository.yml [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+      
+  - type: markdown
+    attributes:
+      value: '## Before you continue, please search our open/closed issues to see if a similar issue has been addressed.'
+
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/meta_repository.yml:13]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/meta_repository.yml [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+      value: '## Before you continue, please search our open/closed issues to see if a similar issue has been addressed.'
+      
+      
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm
+          required: true
+
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/meta_repository.yml:21]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/meta_repository.yml [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+  - type: textarea
+    id: description
+    attributes:
+      label: additions
+      description: what type of additions do you think will make the repository more easily assessible.
+
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/meta_repository.yml:28]:
```suggestion
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Let's see.

Everything except the title (see the comment from earlier that I marked ss unresolved), and some extra lines here and there is fixed now.
I made suggestions on each fix, so you can just click apply changes to commit them, then we're good to go :)

**Comment by Snailedlt** [comment]
The title suggestion: https://github.com/SplitScreen-Me/splitscreenme-discordbot/pull/44#discussion_r901029308

File: .github/ISSUE_TEMPLATE/confiq.yml (hunk 2 above)
Comment (errorline99) [reply .github/ISSUE_TEMPLATE/confiq.yml]: finished it. thank you very much for explaining these stuff to me

File: .github/ISSUE_TEMPLATE/meta_repository.yml [hunk 13]
```diff
@@ -0,0 +1,31 @@
+name: "📇 Meta/Repository Feature"
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
```
Comment (Snailedlt) [inline .github/ISSUE_TEMPLATE/meta_repository.yml:3]:
```suggestion
title: "📇 Meta/Repository Feature"
```

**Review by Snailedlt — APPROVED** [review]
Everything except the title looks good now. I'll just change it so we can merge this in.

Thank you very much for your contribution! 💯

**Review by Snailedlt — APPROVED** [review]
LGTM

**Pull request merged**